odoo/modules/loading.py

```diff
@@ -147,6 +147,9 @@ def load_module_graph(cr, graph, status=None, perform_checks=True,
     models_updated = set()
 
     for index, package in enumerate(graph, 1):
+      with tools.profiler.ExecutionContext({'module': package.name}):
```

Comment:
What about keyword arguments as an alternative API?
```suggestion
      with tools.profiler.ExecutionContext(module=package.name):
```